examples/hello_world/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ find_package(Threads REQUIRED)
 
 set(VSOMEIP_NAME "vsomeip3")
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
```

Comment:
This works, but it's a deprecated way of doing it.  It should be:
```cmake
target_compile_features(<target> PRIVATE cxx_std_17)
```

Reply:
added a fix for this here: https://github.com/COVESA/vsomeip/pull/752